Be more precise about what max/minlength measures (#27659)
mdn/content: files/en-us/web/html/attributes/maxlength/index.md

{{HTMLSidebar}}

> The **`maxlength`** attribute defines the maximum string length the user can enter into an {{htmlelement('input')}} or {{htmlelement('textarea')}}. This must be an integer value 0 or higher. The string [length](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length) is measured in UTF-16 code units, which is usually equivalent to the number of characters for most scripts, although not always — [Strings with length not equal to the number of characters](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#strings_with_length_not_equal_to_the_number_of_characters) has some examples.
> 
> If no maxlength is specified, or an invalid value is specified, the input or textarea has no maximum length.

Reviewer: I love the precision. I have preference for early forward links and having a very short introductory sentence. I also don't feel the need to fully explain where I'm linking to if the likely content of the link can be inferred. This is just to my taste though:

```suggestion
The **`maxlength`** attribute defines the maximum [string length](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length) that the user can enter into an {{htmlelement('input')}} or {{htmlelement('textarea')}}.

The attribute must be an integer value of 0 or higher. The length is measured in UTF-16 code units, which ([for most scripts](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#strings_with_length_not_equal_to_the_number_of_characters)) are equivalent to the number of characters.
If no maxlength is specified, or an invalid value is specified, the `<input>` or `<textarea>` has no maximum length.
```

Reviewer: If you don't agree, just fix up the highlighting

```suggestion
The **`maxlength`** attribute defines the maximum string length the user can enter into an {{htmlelement('input')}} or {{htmlelement('textarea')}}. This must be an integer value of 0 or higher. The string [length](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length) is measured in UTF-16 code units, which is usually equivalent to the number of characters for most scripts, although not always — [Strings with length not equal to the number of characters](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#strings_with_length_not_equal_to_the_number_of_characters) has some examples.

If no `maxlength` is specified, or an invalid value is specified, the `<input>` or `<textarea>` has no maximum length.
```
